src/command/output.rs

```diff
@@ -373,6 +375,10 @@ impl RoverOutput {
                 stderrln!("Readme for {} published successfully", graph_ref,)?;
                 None
             }
+            RoverOutput::PersistedQueriesPublishResponse(response) => Some(format!(
+                "Operation manifest for {} successfully published, creating revision {} of list {}",
```

Comment:
Should that be two sentences or a semicolon? Are there other examples like this?

Reply:
should be two sentences w/newline separator - this is what other commands like this one do